pilot/pkg/serviceregistry/kube/controller/ambient/nodes.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ambient
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"istio.io/api/label"
+	"istio.io/istio/pilot/pkg/util/protoconv"
+	"istio.io/istio/pkg/kube/krt"
+	"istio.io/istio/pkg/workloadapi"
+)
+
+type Node struct {
+	Name     string
+	Locality *workloadapi.Locality
+	Region   string
+	Zone     string
+	Subzone  string
```

Comment:
Remove? We don't set or consume these strings I don't think.

Reply:
d'oh. thanks